R/utils.R

```diff
@@ -907,7 +907,7 @@ compile_functions <- function(env, verbose = FALSE, global = FALSE) {
   invisible(NULL)
 }
 
-expose_functions <- function(function_env, global = FALSE, verbose = FALSE) {
+expose_stan_functions <- function(function_env, global = FALSE, verbose = FALSE) {
```

Comment:
I renamed this _internal_ function to `expose_stan_functions` (the method exposed to users is still named `expose_functions`). Renaming the internal function in `utils.R` let me define an internal function `expose_functions` in `model.R` where I could add the documentation in the same place as the other methods.